site/source/docs/building_from_source/configuring_emscripten_settings.rst

```diff
@@ -29,7 +29,7 @@ The settings file is created the first time a user runs :ref:`emcc <emccdoc>` (o
 	
 Emscripten makes a "best guess" at the correct locations for tools and updates the file appropriately. Where possible it will look for "system" apps (like Python and Java).
 
-The file will probably not include the link to :term:`Fastcomp` (``LLVM_ROOT``) as a manual source build can create this anywhere.
+The file will probably not include the link to LLVM binaries (``LLVM_ROOT``) as a manual source build can create this anywhere.
```

Comment:
This sentence doesn't seem quite right anymore.

How about something like:

"In most cases it is necessary to edit the generated file and modify at  least the `LLVM_ROOT` and `BINARYEN_ROOT` settings point to the correct location of your llvm and binaryen installations respectively."

Reply:
That's much better, thanks.